### PR TITLE
add blogroll, social links

### PR DESCRIPTION
**Pull request by htlcnn**
@hvnsweeting

### CODE REVIEW

File: output/author/htl.html [hunk 1]
```diff
@@ -12,19 +12,19 @@
...
         </header><!-- /#banner -->
 
             <aside id="featured" class="body">
                 <article>
-                    <h1 class="entry-title"><a href="/cai-dat-su-dung-jupyter-notebook.html">Cài đặt, sử dụng Jupyter Notebook</a></h1>
+                    <h1 class="entry-title"><a href="http://blog.fml.vn/cai-dat-su-dung-jupyter-notebook.html">Cài đặt, sử dụng Jupyter Notebook</a></h1>
 <footer class="post-info">
-        <span>T2 06 Tháng sáu 2016</span>
-        	<span>| by <a class="url fn" href="/author/htl.html">htl</a></span>
+        <span>Mon 06 June 2016</span>
```
Comment (hvnsweeting) [inline output/author/htl.html]: why english now?

File: .ropeproject/config.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
+# The default ``config.py``
```
Comment (hvnsweeting) [inline .ropeproject/config.py]: this file should not be here and should in .gitignore

File: output/archives.html [hunk 3]
```diff
@@ -3,7 +3,7 @@
 <head>
         <meta charset="utf-8" />
         <title>FML Blog</title>
-        <link rel="stylesheet" href="/theme/css/main.css" />
+        <link rel="stylesheet" href="http://blog.fml.vn/theme/css/main.css" />
```
Comment (hvnsweeting) [inline output/archives.html]: please don't add output, just add md file, we will render them and put on site

Comment (htlcnn) [inline output/archives.html]: So we only need to push content folder? @hvnsweeting

File: output/author/htl.html (hunk 1 above)
Comment (htlcnn) [inline output/author/htl.html]: It's english when I make html on CentOS (homeserver), vietnamese when I make html on laptop (Ubuntu, regional settings is vietnamese)

**Pull request merged**